CA_DataUploaderLib/VectorDescription.cs

```diff
@@ -7,19 +7,21 @@ namespace CA_DataUploaderLib
     [Serializable]
     public class VectorDescription
     {
-        public List<VectorDescriptionItem> _items;  // only public because we need to Serialize
+        public List<VectorDescriptionItem> _items;  // only public because we need to Serialize  -> Do NOT add or remove anything from this list. 

```

Comment:
its not impossible, but we do need to be **very** careful when doing it if we do a breaking change at some point